python/GafferSceneUITest/SceneGadgetTest.py

```diff
@@ -449,7 +449,11 @@ def testObjectsAtBox( self ) :
 		)
 
 	@unittest.skipIf(
-		os.environ.get( "GAFFER_BUILD_ENVIRONMENT", "" ) == "gcc9",
+		(
+			os.environ.get( "GAFFER_BUILD_ENVIRONMENT", "" ) == "gcc9" or
+			os.name == "nt"
+		),
+		"`objectAt()` fails for `GafferArnoldUITest` on Windows" if os.name == "nt" else
 		"The gcc9 container does not support floating point depth buffers."
```

Comment:
The GCC 9 build container is obsolete now, so lets take the opportunity to remove that part of the check.